src/components/AddTodoForm.jsx

```diff
@@ -15,6 +15,7 @@ function AddTodoForm({
 		setTodoTitle(event.target.value);
 	};
 	const handleAddTodo = (event) => {
+		event.preventDefault();
```

Comment:
This will stop the page from refreshing and messing up any transient state set on any of the components on the page.